src/test/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/ApplicationTest.java

```diff
@@ -41,8 +44,7 @@ void runErrorProcessingTrueTest() {
 
         verify(taskScheduler, times(0)).scheduleWithFixedDelay(eq(mainMessageConsumer), anyLong());
         verify(taskScheduler, times(0)).scheduleWithFixedDelay(eq(retryMessageConsumer), anyLong());
-
-        //ToDo Ensure error consumer is called
+        verify(taskScheduler, times(1)).scheduleWithFixedDelay(eq(errorMessageConsumer), eq(1L));
```

Comment:
if you use an argument matcher like anyXXX for one of the arguments then you have to use eq() to match actual values, that way they are all using matchers - eq and anyXX are both argument matchers. if you are just matching values for all the arguments, then you don't need to use the eq() matchers, just use the values  (errorMessageConsumer, 1L)